### PR TITLE
Corpora comp quick fix

### PR DESCRIPTION
**Pull request by danich1**
This PR is a quick fix for calculating the odds ratio to compare preprints against their published versions. 
On the master branch preprint_published_comparison.png is the same as the global comparison figure. Reason for this is I forgot to update my code to reflect the new changes and I have now fixed that.
No real review necessary as the only main change is the figure files.

### CODE REVIEW

File: biorxiv/corpora_comparison/figure_generator.py [hunk 1]
```diff
@@ -356,7 +356,7 @@
     .append(
         full_plot_df
         .sort_values("odds_ratio", ascending=False)
-        .iloc[:-2]
+        .iloc[:-3]
```
Comment (ben-heil) [inline biorxiv/corpora_comparison/figure_generator.py:359]: Is the `iloc` call here to manually get rid of infinite or NaN odds ratios? If so, would it make more sense to do it explicitly so you aren't manually hiding rows?

**Review by ben-heil — APPROVED** [review]
Looks good, I only left a minor comment for consideration

File: biorxiv/corpora_comparison/corpora_comparison.ipynb [hunk 2]
```diff
@@ -2779,8 +2799,8 @@
    ],
    "source": [
     "preprint_vs_published = get_term_statistics(\n",
-    "    biorxiv_total_count_df, \n",
-    "    pmc_total_count_df, \n",
+    "    preprint_count_df, \n",
```
Comment (ajlee21) [inline biorxiv/corpora_comparison/corpora_comparison.ipynb:2802]: I assume this is where you over-wrote your figure so they looked the same?

**Review by ajlee21 — APPROVED** [review]
LGTM

File: biorxiv/corpora_comparison/figure_generator.py (hunk 1 above)
Comment (danich1) [reply biorxiv/corpora_comparison/figure_generator.py:359]: >Is the iloc call here to manually get rid of infinite or NaN odds ratios? If so, would it make more sense to do it explicitly so you aren't manually hiding rows?

Actually no. The reason I have this line here is to remove non-informative tokens. For example the true leading token in this dataframe is 'u\2009'. This is a unicode for thin space character.  Not informative at all, so it needed to be removed.

File: biorxiv/corpora_comparison/corpora_comparison.ipynb (hunk 2 above)
Comment (danich1) [reply biorxiv/corpora_comparison/corpora_comparison.ipynb:2802]: Correct I accidentally passed the wrong dataframes for this section of code. Hence the duplication.

File: biorxiv/corpora_comparison/figure_generator.py (hunk 1 above)
Comment (ben-heil) [reply biorxiv/corpora_comparison/figure_generator.py:359]: Ahhh I see

**Pull request merged**